stable/airflow/values.yaml

```diff
@@ -236,6 +236,10 @@ scheduler:
   ##
   connections: []
 
+  ## if we remove before adding a connection resulting in a refresh
+  ##
+  refreshConnections: false
```

Comment:
I think the expected behaviour is probably `true`.

Also, as this is a top-level value, please add a description in the readme.md values table.

Reply:
added to the readme.md values table